### DISCUSSION TITLE
What are these warnings? Should I be worried?

### DISCUSSION THREAD

**Discussion by superprohero** [vladmandic/sdnext#855] [Q&A]
![sfaddda](https://github.com/vladmandic/automatic/assets/133063465/f4dabd8d-05e9-4e94-8eca-b5f005709223)

In red, it says, "Warning: Am I missing something? or I am doing something wrong."

**Comment by Tillerz** [comment]
First one you see it's updating it, so next time this message is gone.

Other one: THIS automatic1111 comes with controlnet preinstalled, did you maybe install controlnet manually, on the extensions tab, also?

**Reply by superprohero** [reply]
so does that matter? my stable diffusion is running fine though.

**Reply by vladmandic** [reply]
its running fine because this app is smart enough to ignore duplicate extension. but yes, its a valid warning and you should remove duplicate - just detete it from `/extensions` folder.
